CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence.
+*	@taruss2011, @dnewmanmax
```

Comment:
Didn't know we needed to add this, was it mentioned somewhere?

Reply:
I'm not sure, but I just merged this from master